yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/yb-voyager-pg-gather-assessment-metadata.sh

```diff
@@ -172,6 +185,36 @@ main() {
         fi
     fi
 
+    last_analyze=$(psql $pg_connection_string -tAqc "SELECT schemaname, MAX(last_analyze) FROM pg_stat_all_tables where schemaname=ANY(ARRAY[string_to_array('$schema_list', '|')]) GROUP BY schemaname;")
+
+    null_analyze_schemas=()
+
+    # Read each line
+    while IFS='|' read -r schema ts; do
```

Comment:
I feel like embedding this logic in the SQL query itself might be more easier? (less bash code to read/write) 
AI suggests: 
```
SELECT DISTINCT schemaname 
FROM pg_stat_all_tables 
WHERE schemaname = ANY(ARRAY[string_to_array('$schema_list', '|')])
  AND NOT EXISTS (
    SELECT 1 
    FROM pg_stat_all_tables t2 
    WHERE t2.schemaname = pg_stat_all_tables.schemaname 
      AND t2.last_analyze IS NOT NULL
  );
  ```